BuilderDesign/BuilderDesign/Program.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace BuilderDesign
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var director = new Director();
+            
+            // Margherita Pizza Builder
+            var margheritaPizza = new MargheritaPizzaBuilder();
```

Comment:
It would be better if you used polymorphism to define builders, without creating new builder variable for each one.